sklearn/multiclass.py

```diff
@@ -452,6 +463,7 @@ class OneVsOneClassifier(BaseEstimator, ClassifierMixin, MetaEstimatorMixin):
     def __init__(self, estimator, n_jobs=1):
         self.estimator = estimator
         self.n_jobs = n_jobs
+        self.pairwise_indices_ = None
```

Comment:
I thought we had a consistency check in `test_common` for this kind of things but maybe it's not applied on meta estimators.